lib/propolis/src/hw/uart/lpc.rs

```diff
@@ -15,6 +15,7 @@ struct UartState {
     uart: Uart,
     irq_pin: Box<dyn IntrPin>,
     auto_discard: bool,
+    paused: bool,
```

Comment:
Would be nice to add a little context in a comment here about why we have `paused` (to paper over chardev shortcomings around save/restore), so it's clear if/when we circle back to fix that stuff.

Reply:
added something in e7fe716